### PR TITLE
ZH-trans: add generated/...

### PR DESCRIPTION
**Pull request by xichengliudui**
ZH-trans: add generated/...

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
should those files be auto-generated? @tengqm  @markthink

**Comment by tengqm** [comment]
yes, @zhangxiaoyu-zidif they are auto-generated *for English version*. Until the day we have upstream support to i18n for `kube*` components, manual translation is the only option here.

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_etcd-ca.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+
+<!--
+Generates the self-signed CA to provision identities for etcd
+-->
+生成自签名证书以为 etcd 提供标识
```
Comment (Mr-Linus) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_etcd-ca.md]:
```suggestion
 生成自签名证书以便为 etcd 提供标识
```
是不是改成以便要好一点

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_etcd-ca.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<!--
+If both files already exist, kubeadm skips the generation step and existing files will be used. 
+-->
+如果两个文件都已存在，kubeadm 将跳过生成步骤，并将使用现有文件。
+
+<!--
+Alpha Disclaimer: this command is currently alpha.
+-->
+免责声明：此命令目前属于 alpha。
```
Comment (Mr-Linus) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_etcd-ca.md]:
```suggestion
Alpha 免责声明：此命令目前属于 alpha。
```

**Comment by markthink** [comment]
@zhangxiaoyu-zidif 虽然原始英文是自动生成的，但目前中文自动生成还是不现实..

**Pull request merged**